test/UtilsTest.php

```diff
@@ -240,19 +241,23 @@ public function testDumpConfig()
 		$this->assertEquals($sContent, Utils::DumpConfig());
 	}
 
-	public function testCheckModuleInstallation()
+	public function PrepareCheckModuleInstallation($expectedCallCount=1) : RestClient
```

Comment:
Better to be clear/strict
```suggestion
	protected function PrepareCheckModuleInstallation(int $iExpectedCallCount) : RestClient
```